Shared/Prefs.swift

```diff
@@ -14,6 +14,7 @@ public struct PrefsKeys {
     public static let KeyMailToOption = "MailToOption"
     public static let HasFocusInstalled = "HasFocusInstalled"
     public static let HasPocketInstalled = "HasPocketInstalled"
+    public static let HasLeanplumStarted = "HasLeanplumStarted"
```

Comment:
This is created but not used anywhere I can see?

Reply:
Yeah this should not be a pref key. It can be determined at runtime, and is not stored in prefs.